presentation/src/main/java/com/architect/coders/mu8/comics/ComicsActivity.kt

```diff
@@ -34,14 +30,9 @@ class ComicsActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_comics)
 
-        toolbar = findViewById(R.id.comic_activity_toolbar)
-        setSupportActionBar(toolbar)
-        supportActionBar?.setDisplayShowTitleEnabled(false)
-
-        toolbarTitle = findViewById(R.id.toolbar_title)
-        toolbarTitle.text = getString(R.string.comics_name)
+        val binding: ActivityComicsBinding =
+            DataBindingUtil.setContentView(this, R.layout.activity_comics)
```

Comment:
I think this fits in one line

Reply:
Done

Reply:
Done

Reply:
Its not solved, please nove it in single line.

Reply:
Done